evennia/accounts/accounts.py

```diff
@@ -236,6 +236,39 @@ def characters(self):
 
         return objs
 
+    def add_character_to_playable_list(self, character: "DefaultCharacter"):
+        """
+        Add a character to this account's list of playable characters.
+        """
```

Comment:
It's worth to expand the docstring here, since this list is not really limiting what can be played (the `puppet:` lock does). The list is only used as a quick way to show a list on the character-selection screen. Not to mention that this listing is not unique - any number of people can have a character in their playable lists.